client/my-sites/domains/domain-management/domain-connect-mapping/index.jsx

```diff
@@ -47,7 +47,8 @@ class DomainConnectMapping extends React.Component {
 	isDomainConnectComplete = () => {
 		const queryObject = parse( window.location.search.replace( '?', '' ) );
 		const status = get( queryObject, 'status', null );
-		return status === this.getRedirectUriStatusString();
+		const redirectUriStatusString = this.getRedirectUriStatusString();
```

Comment:
The name of the function isn't really status only now. I'd also use `get( queryObject, redirectUriStatusString, false )`. It took me a while to figure what is happening.

Reply:
But it _is_ still the status string. It's the string that lets us know the completeness status of the template application. It's just that it's not always the value of the status variable in the query string.

Also, if I use `get`, I just get an empty string since `redirectUriStatusString` is a key rather than a value. And I really just want to know if the key is present in the query string in this case.

I expect this to be temporary since the spec considers this a malformed status response.

Reply:
Yy, but what I meant is instead of using the `something in queryObject`, use `get()`. :)

Reply:
But my point is that `get()` will just return an empty string in this case, so I'd have to either check for something like `false !== something` or `"" == something`. Both of which seem more confusing to me than `key in object` which is a standard javascript expression to check for the presence of a key.

Maybe I'm still missing something though. :-/

Reply:
You sure? If no key, it returns the default. And it's an expression we are used to now. But your call! Already approved. :)

Reply:
You're right. `in` is better. I guess the main reason for my confusion was the name `redirectUriStatusString`. But yeah, temporary. Thanks for discussing it. :)